addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -8,6 +8,36 @@ function factory(dependencies) {
 
     class ChatWindow extends dependencies['mail.model'] {
 
+        /**
+         * @override
+         */
+        static create(data) {
+            const chatWindow = super.create(data);
+            chatWindow._onShowHomeMenu.bind(chatWindow);
+            chatWindow._onHideHomeMenu.bind(chatWindow)
+
+            chatWindow.env.messagingBus.on(
+                'hide_home_menu',
+                chatWindow,
+                chatWindow._onHideHomeMenu
+            );
+            chatWindow.env.messagingBus.on(
+                'show_home_menu',
+                chatWindow,
+                chatWindow._onShowHomeMenu
+            );
```

Comment:
Not really a mistake, but `this.env` should be available also on static methods I think.